tx_list.go

```diff
@@ -139,15 +165,16 @@ func (tx *Tx) LSize(bucket string, key []byte) (int, error) {
 // 1 being the next element and so on.
 // Start and end can also be negative numbers indicating offsets from the end of the list,
 // where -1 is the last element of the list, -2 the penultimate element and so on.
-func (tx *Tx) LRange(bucket string, key []byte, start, end int) (list [][]byte, err error) {
+func (tx *Tx) LRange(bucket string, key []byte, start, end int) (lists [][]byte, err error) {
```

Comment:
Why change the name lists？

Reply:
because the name has conflict， I can solve it

Reply:
I solve it in my latest merge request